src/pages/gl-integration-logic/index.js

```diff
@@ -0,0 +1,133 @@
+import { useContext } from 'react'
+import toast from 'react-hot-toast'
+import Table from 'src/components/Shared/Table'
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useResourceQuery } from 'src/hooks/resource'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { ControlContext } from 'src/providers/ControlContext'
+import { GeneralLedgerRepository } from 'src/repositories/GeneralLedgerRepository'
+import IntegrationLogicWindow from './Windows/IntegrationLogicWindow'
+import { useWindow } from 'src/windows'
+
+const GLIntegrationLogic = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack } = useWindow()
+
+  async function fetchWithSearch({ qry }) {
+    const response = await getRequest({
+      extension: GeneralLedgerRepository.IntegrationLogic.snapshot,
+      parameters: `_filter=${qry}`
+    })
+
+    return response
+  }
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50 } = options
+    try {
+      const response = await getRequest({
+        extension: GeneralLedgerRepository.IntegrationLogic.page,
+        parameters: `_startAt=${_startAt}&_pageSize=${_pageSize}&filter=`
```

Comment:
please remove pagination params ( pageSize and startAt ) , they are not requested by api